### DISCUSSION TITLE
How to visualize experiment's results?

### DISCUSSION THREAD

**Discussion by artizzq** [GeoChemFoam/GeoChemFoam#5] [Q&A] [answered]
My system is Linux Mint 21 Cinnamon running inside VirtualBox. The latest GeoChemFoam 4.8 Docker image was used.

So I followed up the guidelines in Tutorial 5 https://github.com/GeoChemFoam/GeoChemFoam/wiki/Test-Case-05-Bentheimer-heat-transfer.

And got those files generated (in this screenshot I put them into OpenFOAM, info for that below):
![image](https://user-images.githubusercontent.com/56273839/189470652-ace081d0-c03e-4e06-9861-e0ba16449e6d.png)

I have tried to visualize them using "paraFoam &" command inside your GeoChemFoam's Docker image but it says that paraview not found.
![image](https://user-images.githubusercontent.com/56273839/189470977-734e8e87-b774-4028-b7e4-673b99bf762c.png)

Ok, then I decided to download OpenFOAM and I copied the result there. As a result of running the command "paraFoam &", ParaView opens but when I try to use "Apply Button" on foam object with selected "T" field only (I need to visualize a result of steady-state temperature equation) I dont see anything changing in a viewport.
![image](https://user-images.githubusercontent.com/56273839/189471091-c6bb4984-659c-4e9f-959f-17a49c089abd.png)

So what am I doing wrong and how am I able to see the results I made? 

I havent seen anything about visualization in your documentation or maybe I am wrong? 

Please point me how to see the result. I am pretty beginner and thats a challenge for me.

**Comment by hannahmenke** [comment]
Hi, 

Yes, paraview can have quite a learning curve but there are a few tricks. 

If you copy over the entire Bentheimer directory to your desktop. Then start paraview and drag the Paraview_Loader.foam file into paraview. Then uncheck 'skip zero time' and then apply and refresh it should load the temperature results. Let me know if this does not work.

**Comment by artizzq** [comment]
Same thing however :(
![image](https://user-images.githubusercontent.com/56273839/189486756-f0750238-b579-4c2f-a2fe-d4e94e0f2919.png)

**Comment by GeoChemFoam** [comment]
Can you select T rather than solid color at the top?

On Sep 10, 2022, at 14:59, artizzq ***@***.***> wrote:

﻿
****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Same thing however :(
[image]<https://user-images.githubusercontent.com/56273839/189486756-f0750238-b579-4c2f-a2fe-d4e94e0f2919.png>

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/5#discussioncomment-3614590>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFROYAMSPUIWM7QYCV3V5SH3ZANCNFSM6AAAAAAQJGCWZQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.

**Comment by hannahmenke** [comment, marked as answer]
First check that the simulation has run correctly. You should have these files:

<img width="477" alt="Screenshot 2022-09-10 at 15 46 06" src="https://user-images.githubusercontent.com/42167513/189488890-ffe22efe-75d0-4253-82c2-9ab3669d635b.png">

Then drag the Paraview_loader.foam into paraview. Uncheck skip zero time and apply and refresh:
<img width="1679" alt="Screenshot 2022-09-10 at 15 47 05" src="https://user-images.githubusercontent.com/42167513/189488908-8e5a832d-6958-4697-ab49-abfbd560c5d3.png">

Select T from the dropdown menu:
<img width="464" alt="Screenshot 2022-09-10 at 15 58 09" src="https://user-images.githubusercontent.com/42167513/189489041-8b0b8fcc-0e8a-4966-9d42-df1096eda5ed.png">


Clip the constant temperature boundary cells off to see the inside of the Bentheimer
<img width="1602" alt="Screenshot 2022-09-10 at 15 53 45" src="https://user-images.githubusercontent.com/42167513/189488995-f9d4e403-cb46-4dd8-a598-bc70ba2fb074.png">


Hope this helps

**Comment by artizzq** [comment]
❤️🤗 [hannahmenke](https://github.com/hannahmenke), ❤️🤗 [GeoChemFoam](https://github.com/GeoChemFoam), my deepest hug to you! Thank you very much! It works now! I didnt know about this trick! I have to deep more about ParaView's functionality.
![image](https://user-images.githubusercontent.com/56273839/189493327-f2ec69f2-d796-4673-8b55-d820b6d52366.png)
